### DISCUSSION TITLE
where I can see the trends of I following people

### DISCUSSION THREAD

**Discussion by MidhiaQueiroz** [community/community#67523] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Please tell me where I can see the trends of I following people.

I try any filters, but the result is not I want.

**Comment by MidhiaQueiroz** [comment]
.

**Comment by fhk** [comment]
yes, this used to be the default and now its gone WTF!?

**Comment by mehulrajdev** [comment, marked as answer]
You cannot anymore, they have changed the feed. Now you should follow what GitHub wants you to follow.
